src/OSFramework/DataGrid/Configuration/Column/ColumnConfig.ts

```diff
@@ -23,6 +23,7 @@ namespace OSFramework.DataGrid.Configuration.Column {
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         public editor: any;
         public errorMessage: string;
+        public externalURL: string;
```

Comment:
I think we can remove this parameter from here and only maintain on the ColumnConfigAction Class only since this is not common to all columns.

Reply:
Done